README.md

```diff
@@ -1,19 +1,19 @@
 # Keeweb for YunoHost #
 
-Web client for reading and editing Keepass files locally. It can also sync with Owncloud, Google Drive and Dropbox.
+Web client for reading and editing Keepass files locally. It can also sync with WebDAV (Owncloud, Nextcloud...), Dropbox, Google Drive, OneDrive...
 ![screenshot](https://habrastorage.org/files/ec9/108/3de/ec91083de3e64574a504bc438d038dec.png)
 
-**Current version: 1.1.4**
-
 ## Usage with OwnCloud ##
 1. Open your file through webdav using https://linktoowncloud/remote.php/webdav and your username and password
 
 ## Usage for Dropbox sync ##
 1. [create](https://www.dropbox.com/developers/apps/create) a Dropbox app
 2. find your app key (in Dropbox App page, go to Settings/App key)
-3. Enter the app key during install
-
+3. Enter the app key when prompted
 
+## Add default files on your homepage ##
+1. Read https://github.com/keeweb/keeweb/wiki/Configuration#json-app-config
+2. In your config.json add the "files" entry with the appropriate settings
```

Comment:
custom updates of config.json will be overwritten on every app update, so maybe this `files` setting should be a manifest argument ? ( I don't know keeweb at all, It's just a thougt while reading this PR)